pycbc/psd/__init__.py

```diff
@@ -82,7 +82,11 @@ def from_cli(opt, length, delta_f, low_frequency_cutoff,
     if (opt.psd_model or opt.psd_file or opt.asd_file):
         # PSD from lalsimulation or file
         if opt.psd_model:
-            psd = from_string(opt.psd_model, length, delta_f, f_low)
+            if hasattr(opt, 'fake_strain_extra_args'):
+                psd = from_string(opt.psd_model, length, delta_f, f_low,
+                                  opt.fake_strain_extra_args)
```

Comment:
This should be
(1) Why do we need an if statement here? Can you set a default of an empty dictionary?

(2)
```
**opt.fake_strain_extra_args)
```